source/_guides/2-create-an-organization.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Create an Organization
+use: [guides]
+---
+
+## Organizations own Images
+
+Images uploaded to Rokka are always owned by an organization, not an individual user. When you create an organization, you automatically become its administrator and can add other users with a set of permissions. Users are identified by their email address.
+
+## Create a new Organization
+
+To get started, lets create an organization:
+
+```bash
+curl -X PUT 'https://api.rokka.io/organizations/testorganization' -d '{
+    'billing_email' => 'billing@testorganization.com',
+    'display_name' => 'Test Organization Ltd.'
+}'
+```
+```php
+/** @var \Rokka\Client\User $userClient */ 
+$userClient->createOrganization('testorganization', 'billing@testorganization.com', 'Test Organization Ltd.');
+```
+
+This creates an organization called "testorganization" and returns a json structure that reflects the submitted data and adds the unique id of your organization. 
+
+```javascript
+{
+    "id": "c03683b067927d77973b458e0baa40aa7b5e5418",
```

Comment:
UUIDs are internal identifiers only, needed for linking users to organizations. Can also be helpful for debugging.